### DISCUSSION TITLE
Need multiple root nodes

### DISCUSSION THREAD

**Discussion by xahodo** [freeplane/freeplane#1180] [Help]
I'm trying to use freeplane to help create a novel (first steps). In doing so, I found that I actually need multiple root nodes (characters, locations, science, etc.), in order to keep things organized.

This culminated in me being curious as to how I might achieve what I want. As it stands I'm baffled how I'd achieve something like that.

Any suggestions?

**Comment by macmarrum** [comment]
A mindmap has a single root node. This is a rule.

To achieve what you described, I'd use either 
1. virtual root nodes (view root) and switch view to one of them
2. a master mindmap with each of your "multiple root nodes" moved to a new mindmap

I'd start with something like this

![novel](https://github.com/freeplane/freeplane/assets/7197131/dec7ca87-724b-4e45-b510-a08b93803d47)

and then
1. select e.g. Characters and do `Navigate->Jump in`
2. select e.g. Characters and do `File->Move branch to new map…`, then use the link on the node to switch to the new Characters mindmap

**Comment by EdoFro** [comment]
you can also:
- use free positioning nodes and use the new node layouts to simulate multiple mind maps into one. 
- use multiple map views and using the jump in method

Personally I would prefer to do your first approach in one map and only if it gets to big then I would split into parts. 

in a novel you have ‘events’, ‘characters’, etc and they are interrelated. So you will use some kind of links between the nodes of each of the branches, maybe connectors. They only work in the same map.

My approach is always to center my work in the information first and don’t worry about formatting. 
If you use the “normal” layout, Freeplane will keep things clean and formatted. If you start placing nodes in special positions and worry about the format of the map you will have problems with overlapping nodes. At the end you will restrict your creative process by how the map looks like. 

jm2c
edo
